src/main/java/ai/levo/ConfigMenu.java

```diff
@@ -33,7 +34,13 @@ public class ConfigMenu implements Runnable, IExtensionStateListener {
     /**
      * Default value of the Levo's Satellite URL.
      */
-    public static volatile String DEFAULT_LEVO_SATELLITE_URL = "http://localhost:9999";
+    public static volatile String DEFAULT_LEVO_SATELLITE_URL = "https://collector.levo.ai";
```

Comment:
@mgiamberardino might be cleaner to have this as final constant and have separate variable to hold the value.